cypress/integration/mytools.js

```diff
@@ -5,6 +5,34 @@ describe('Dockstore my tools', function() {
      cy.visit(String(global.baseUrl) + "/my-tools")
   });
 
+  describe('Should contain extended DockstoreTool properties', function() {
+    it('visit another page then come back', function() {
+      cy.get('a').contains('Tools').click()
+      cy.contains('b3')
+      cy.get('a').contains('My Tools').click()
+      cy
+      .get('accordion')
+        .children(':nth-child(2)')
+        .click()
+        .get('.panel')
+        .children(':nth-child(2)')
+        .contains('a', 'b1')  
+        .click()
+        cy.contains('GitHub')
+        cy.contains('https://github.com/A2/b1')
+        cy.contains('Quay.io')
+        cy.contains('quay.io/A2/b1')
+        cy.contains('Last Build')
+        cy.contains('Mar. 15 2016 at')
```

Comment:
Skip the dates, they're likely to change due to #514

Reply:
Skip the values only or also 'Last Build' and 'Last Updated'?

Reply:
Maybe those too